Formula/aurora-cli.rb

```diff
@@ -11,7 +11,8 @@ class AuroraCli < Formula
     sha256 "9b45a0ba82e22ff2e272c80977d047a16b5164edb0efc198ffc90ede170c5444" => :mavericks
   end
 
-  depends_on :java => "1.8+"
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!